### PR TITLE
[WIP] add support for Symfony3

### PR DESCRIPTION
**Pull request by rande**

### CODE REVIEW

File: Admin/Admin.php [hunk 1]
```diff
@@ -1240,7 +1241,7 @@ public function hasRoute($name)
     /**
      * {@inheritdoc}
      */
-    public function generateObjectUrl($name, $object, array $parameters = array(), $absolute = false)
+    public function generateObjectUrl($name, $object, array $parameters = array(), $absolute = UrlGeneratorInterface::ABSOLUTE_PATH)
```
Comment (Soullivaneuh) [inline Admin/Admin.php]: What is the value of this constant?

File: Controller/CRUDController.php [hunk 2]
```diff
@@ -1364,4 +1364,12 @@ private function resolveRequest(Request $request = null)
 
         return $request;
     }
+
+    /**
+     * @return Request
+     */
+    public function getRequest()
+    {
+        return $this->container->get('request_stack')->getCurrentRequest();
```
Comment (Soullivaneuh) [inline Controller/CRUDController.php]: Does this work on 2.3?

Comment (rande) [inline Controller/CRUDController.php]: request_stack is SF2.4+ , so no...

Comment (Soullivaneuh) [inline Controller/CRUDController.php]:
```php
if ($this->container->has('request_stack')) {
    return $this->container->get('request_stack')->getCurrentRequest();
}

return $this->container->get('request');
```

File: Admin/Admin.php (hunk 1 above)
Comment (rande) [inline Admin/Admin.php]: not false on latest version of symfony

Comment (Soullivaneuh) [inline Admin/Admin.php]: Well, as I can see, you just pass the value to the router. Should be BC AFAIK.

But, does `ABSOLUTE_PATH` constant exist?

Comment (rande) [inline Admin/Admin.php]: yes, the constant exist from 2.3 to 3.0

Comment (Soullivaneuh) [inline Admin/Admin.php]: :+1:

**Pull request closed without merging**